README.md

```diff
@@ -2,40 +2,55 @@
 Let's make a planetary system!
 
 ## Learning Goals
-### Wave 1
-- Create a custom **class**
-- Create and use an **initialize** method in your class
-- Create and use **instance variables** in your class
 
-### Wave 2
-- Use an **array** to store a list of objects
-- Create and use a **method**
+- Practice creating Custom classes
+- Use a hash to initialize an object
+- Create an object which contains a collection of other objects
+- Practice creating instance methods
+
 
 ## Baseline
```

Comment:
Traditionally I think we've used the term "baseline" to refer to mostly boilerplate setup stuff that didn't involve real implementation of any logic -- and which we wanted to ensure everyone had completed before anyone moved on to starting with Wave 1.

In light of that, do we want to consider the instructions within this section to be part of the baseline or should they be wave 1 (and have other waves adjusted accordingly)?

Looping in @kariabancroft @PilgrimMemoirs @droberts-ada because this seems like a broader question about defining a standard which we should adhere to with all project definitions.